kie-wb-parent/pom.xml

```diff
@@ -81,6 +249,55 @@
       <modules>
         <module>kie-wb-home-page-product</module>
       </modules>
+
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>org.codehaus.mojo</groupId>
+              <artifactId>gwt-maven-plugin</artifactId>
+              <configuration>
+                <compileSourcesArtifacts combine.children="append">
+                  <!-- Product Theme -->
+                  <compileSourcesArtifact>org.kie:kie-wb-theme-product</compileSourcesArtifact>
+                  <!-- Product Home Page -->
+                  <compileSourcesArtifact>org.kie:kie-wb-home-page-product</compileSourcesArtifact>
+                  <compileSourcesArtifact>org.kie.workbench.screens:kie-wb-common-home-api</compileSourcesArtifact>
+                  <compileSourcesArtifact>org.kie.workbench.screens:kie-wb-common-home-client</compileSourcesArtifact>
+                </compileSourcesArtifacts>
+              </configuration>
+            </plugin>
+          </plugins>
+        </pluginManagement>
+      </build>
+    </profile>
+
+    <!-- Full profile builds all GWT permutations -->
+    <profile>
+      <id>fullProfile</id>
+      <activation>
+        <property>
+          <name>full</name>
+        </property>
+      </activation>
+      <properties>
+        <gwt.memory.settings>-Xmx4g -Xms1g -Xss1M</gwt.memory.settings>
+        <gwt.compiler.localWorkers>4</gwt.compiler.localWorkers>
```

Comment:
done, removed the custom settings from the fullProfile. reusing the same properties for the different profiles now.